dist/js/medium-editor.js

```diff
@@ -1832,6 +2013,53 @@ var Events;
         }
     };
 
+    // Do feature detection to determine if the 'input' event is supported correctly
+    // Currently, IE does not support this event on contenteditable elements
+
+    var tempFunction = function () {
+            Events.prototype.InputEventOnContenteditableSupported = true;
+        },
+        tempElement,
+        existingRanges = [];
+
+    // Create a temporary contenteditable element with an 'oninput' event listener
+    tempElement = document.createElement('div');
+    tempElement.setAttribute('contenteditable', true);
+    tempElement.innerHTML = 't';
+    tempElement.addEventListener('input', tempFunction);
+    tempElement.style.position = 'absolute';
+    tempElement.style.left = '-100px';
+    tempElement.style.top = '-100px';
+    document.body.appendChild(tempElement);
```

Comment:
Since this modification, I have to wrap my `new MediumEditor('.editable')` inside a `onload` otherwise the `document.body` is null :confused: 

As far as I understand, this `oninput` was made only for IE right? Shouldn't we test if we are on IE and not attach this element to the body?

In a quick and drity mode I commented this line and 2053 and everything works again.